cytominer_eval/operations/enrichment.py

```diff
@@ -0,0 +1,78 @@
+"""Function to calculate the enrichtment score for a given similarity matrix.
+"""
+import numpy as np
+import pandas as pd
+from typing import List
+import scipy
+
+from .util import assign_replicates, calculate_grit, check_grit_replicate_summary_method
+from cytominer_eval.transform.util import (
+    set_pair_ids,
+    set_grit_column_info,
+    assert_melt,
+)
+
+
+def enrichment(
+    similarity_melted_df: pd.DataFrame, replicate_groups: List[str], percentile: float,
```

Comment:
Just checking, is it intentional that `percentile` doesn't specify a default?

Reply:
set it to 0.9